client/settings/payments-and-transactions-section/index.js

```diff
@@ -17,6 +19,19 @@ import {
 import InlineNotice from 'wcstripe/components/inline-notice';
 import UpeToggleContext from 'wcstripe/settings/upe-toggle/context';
 
+const TooltipBankStatementHelp = () => (
+	<Tooltip
+		content={ __(
+			'Bank statement must consist of at least 1 Latin letter and cannot contain special characters.',
+			'woocommerce-gateway-stripe'
+		) }
+	>
+		<span style={ { flexShrink: 0 } }>
```

Comment:
Do we need the `flexShrink` as the parent is not a flex container?

Reply:
Good find. It was not needed.